<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="start_Button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>변환시작</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_Button">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>닫기</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>웹 교회관리용 파일 만들기</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>102</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>변환 시작일자</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>변환 종료일자</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="start_dateEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="correctionMode">
    <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::DaySection</enum>
   </property>
   <property name="date">
    <date>
     <year>2022</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QDateEdit" name="end_dateEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="correctionMode">
    <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::DaySection</enum>
   </property>
   <property name="date">
    <date>
     <year>2022</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>start_dateEdit</tabstop>
  <tabstop>end_dateEdit</tabstop>
  <tabstop>start_Button</tabstop>
  <tabstop>close_Button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>close_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>start_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>change_start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>change_start()</slot>
 </slots>
</ui>
